pkg/internal/controllers/certificaterequests.go

```diff
@@ -142,12 +146,35 @@ func addCertificateRequestController(ctx context.Context, opts Options) error {
 // function will call the approver manager to evaluate whether a
 // CertificateRequest should be approved, denied, or left alone.
 func (c *certificaterequests) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
+	result, patch, resultErr := c.reconcileStatusPatch(ctx, req)
+	if patch != nil {
+		con, patch, err := ssa_client.GenerateCertificateRequestStatusPatch(req.Name, req.Namespace, patch)
+		if err != nil {
+			err = fmt.Errorf("failed to generate connection patch: %w", err)
```

Comment:
Typo in error message.

Should say "failed to generate CertificateRequest.Status patch.